cmake/find/FindGBenchmark.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2015-2016, University College London
```

Comment:
Let me see if the maintainers of the code will take a fix to their build.
This pull-request will be on hold until then.